test/t_second_doc

```diff
@@ -23,7 +23,7 @@ EOF
 
 set -x
 echo -n > $T.out
-test_run $REQFLOW stat -s -c $T.req
-test_run $REQFLOW trac -c $T.req
+test_run ! $REQFLOW stat -s -c $T.req
+test_run ! $REQFLOW trac -c $T.req
```

Comment:
I add that using ! in a shell script is error prone, as ! has a special meaning in under some condition.

Reply:
Yes but we are precisely in the case where we expressly need this special meaning.
But I am open to any less error prone way if you know one.